WcaOnRails/app/controllers/delegates_panel_controller.rb

```diff
@@ -28,6 +30,16 @@ def update_crash_course
     end
   end
 
+  def pending_claims_for_subordinate_delegates
+    # Show pending claims for a given user, or the current user, if they can see them
+    @user = User.includes(subordinate_delegates: [:users_claiming_wca_id]).find_by_id(params[:user_id] || current_user.id)
```

Comment:
nit: I think this should be a `find_by_id!` so we get a 404. As written I think `@user` will end up `nil`, and it looks like it goes on to crash the page (try going to https://www.worldcubeassociation.org/delegate/pending-claims/23432234234234324324324234322 to see what I mean)

Reply:
Also, I think this would allow senior delegates to view *other* senior delegates's claims, which honestly seems fine to me.